### PR TITLE
Refine getting started guide to support own GitLab instance

### PR DESCRIPTION
**Pull request by tobru**
This PR refines the getting started guide to help with using it with an own GitLab instance. Plus it fixes some small annoyances and actually makes Commodore work again with the latest Commodore image (which is now based on Poetry).

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -81,33 +83,36 @@ Check that the API is accessible:
...
 
-Replace `MYTOKEN` with the generated GitLab.com API token.
+Replace `MYTOKEN` with the generated GitLab API token. If you're using your own GitLab instance, amend `GITLAB_ENDPOINT` and `GITLAB_SSH_HOST_KEY` (You can use the `ssh-keyscan` tool to get the SSH host key of your own GitLab instance).
 
 [source,shell]
 ----
-export GITLABCOM_TOKEN=MYTOKEN
+export GITLAB_TOKEN=MYTOKEN
+export GITLAB_ENDPOINT=gitlab.com
+export GITLAB_SSH_HOST_KEY="gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf"
```
Comment (srueg) [inline docs/modules/ROOT/pages/getting-started.adoc:111]: As mentioned before; I fear Argo CD requires RSA host keys to function properly (not 100% sure though)

Comment (srueg) [inline docs/modules/ROOT/pages/getting-started.adoc:111]:
```suggestion
export GITLAB_SSH_HOST_KEY=$(ssh-keyscan -t rsa $GITLAB_ENDPOINT)
```

**Review by srueg — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/getting-started.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/getting-started.adoc:111]: Yeah, this will be fixed in #64

Comment (tobru) [reply docs/modules/ROOT/pages/getting-started.adoc:111]: Will be part of #64

**Review by srueg — APPROVED** [review]

**Pull request merged**